crates/sui-graphql-rpc/src/types/epoch.rs

```diff
@@ -306,3 +331,48 @@ impl Epoch {
         }))
     }
 }
+
+#[async_trait::async_trait]
+impl Loader<EpochKey> for Db {
+    type Value = Epoch;
+    type Error = Error;
+
+    async fn load(&self, keys: &[EpochKey]) -> Result<HashMap<EpochKey, Epoch>, Error> {
+        use epochs::dsl;
+
+        let epoch_ids: BTreeSet<_> = keys.iter().map(|key| key.epoch_id as i64).collect();
+        let epochs: Vec<QueryableEpochInfo> = self
+            .execute_repeatable(move |conn| {
```

Comment:
Thoughts on having a comment explaining why we chose to filter checkpoint on application side instead of in the sql query? Might not be immediately obvious to others